### ISSUE TITLE
[Bug] Economy Plugin Glitch

### ISSUE THREAD

**Issue by docwhomc** [EarthMC/Issue-Tracker#753]
**Server:** Towny and Factions

**Describe the bug**
An error occurs when using `/f bank` or `\t bank` commands, resulting in loss of gold.

**To Reproduce**
Steps to reproduce the behavior (on factions):
1. Do `/f bank deposit <amount>` while holding gold.  -->  Get error message.
2. Check faction's balance with `/f who`.  -->  Faction balance unchanged (still "no balance").
3. Try to withdraw gold with `/f bank withdraw <amount>`  -->  Same error message as in 1.

**Expected behavior**
Gold gets deposited into bank instead of destroyed.

**Comment by Subking** [comment]
I've had this same problem also but in towny, I lost all the gold that I had deposited into my bank when the bug that gave people the "[Towny] null" error which was yesterday. I've already made a ticket and everything and several modders have already contacted me but no one has done anything about it and I was already assured that it's something that can be fixed but I have been waiting for several hours already waiting to recover my lost gold.

**Comment by Subking** [comment]
So this was when the bug that caused the "[Towny] null" was still occurring and I was also one of the people that the bug affected and I was still figuring out the /t controls since I had just gotten my town, anyways, I made one deposit of 17 gold if I remember correctly and I had gotten an error but my gold was gone so I thought the error was an error itself and I still managed to deposit my gold. Then I made another deposit of 12 gold and the same thing happened and I thought nothing of it since my gold was gone and I just assumed it was already in my town bank. I couldn't access my /t commands because of that bug so I couldn't withdraw either. I thought my gold would be fine til' they fixed the bug and when I logged in today and tried to withdraw was when I realized that indeed all the gold I had deposited was gone.
